### ISSUE TITLE
Excuse me. I want to ask about the problem of setting up a Chinese language pack.

### ISSUE THREAD

**Issue by zhouxiangbin** [gcobb321/icloud3_v3#340]
By default, iCloud3 is in English. I found this in the folder ~/custom_components/icloud3/translations/zh-Hans.json. May I ask how I can set it up to display Chinese? Thank you very much for guiding me. I am extremely grateful!

**Comment by lyfqdl** [comment]
同样的问题，感谢。

**Comment by lyfqdl** [comment]
@zhouxiangbin 最新测试版icloud3.v3.1b5.zip，你测试了吗？
我的无法登陆，一直提示icloud_acct_not_logged_into

**Comment by lyfqdl** [comment]
@zhouxiangbin 最新测试版icloud3.v3.1b5.zip，你测试了吗？
我的无法登陆，一直提示icloud_acct_not_logged_into

**Comment by lyfqdl** [comment]
Can you provide Chinese? I really appreciate it．
I put this file in \icloud3\translations, and iCloud3 v3.1 will not load and will not work. @gcobb321

**Comment by gcobb321** [comment]
No, I don’t know Chinese. Another user provided a translation file for v3.0 but there were a lot of changes in v3.1 so I removed it. I am still changing it trying to get v3.1 settled down. If you want to make one, I will add it to a release or set up a translated repository on the GitHub for iCloud3

**Comment by gcobb321** [comment]
Closing due to no activity and the release of iCloud3 v3.1.5
